<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>586</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>586</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="option_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>104</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QPushButton" name="open_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="query_account_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
     <widget class="QPushButton" name="daily_turnover_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>115</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Turnover</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="welcome_page">
      <widget class="QLabel" name="welcome_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>511</width>
         <height>471</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;welcome to management system&lt;/p&gt;&lt;p&gt;push button to finish function&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="opening_page">
      <widget class="QComboBox" name="type_comboBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>310</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Save</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Save</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Credit</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="opening_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>521</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Opening Account:</string>
       </property>
      </widget>
      <widget class="QLabel" name="account_input_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input Account:</string>
       </property>
      </widget>
      <widget class="QLabel" name="password_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input Password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="choose_type_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select Type:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="account_input_lineEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>120</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_input_lineEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="set_pushButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>400</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="query_page">
      <widget class="QLabel" name="query_account_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input Query Account:</string>
       </property>
      </widget>
      <widget class="QLabel" name="query_turnover_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>138</y>
         <width>617</width>
         <height>36</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>741</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Turnover:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_accout_lineEdit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="query_pushButton">
       <property name="geometry">
        <rect>
         <x>524</x>
         <y>20</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Query</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="turnover_tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>617</width>
         <height>361</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>408</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="account_info_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>617</width>
         <height>62</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>741</width>
         <height>67</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="account_label">
          <property name="maximumSize">
           <size>
            <width>54</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Account:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="account_show_label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="type_label">
          <property name="maximumSize">
           <size>
            <width>114</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="type_show_label">
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="balance_label">
          <property name="maximumSize">
           <size>
            <width>52</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Balance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="balance_show_label">
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="daily_turnover_page">
      <widget class="QTableWidget" name="daily_turnover_showtableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>84</y>
         <width>617</width>
         <height>471</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100000</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="money_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>80</height>
        </rect>
       </property>
       <widget class="QLabel" name="total_money_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Total Money:</string>
        </property>
       </widget>
       <widget class="QLabel" name="total_money_show_label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="save_money_show_label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="save_show_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Saving Money:</string>
        </property>
       </widget>
       <widget class="QLabel" name="get_money_show_label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="get_money_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Get Money:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="select_date_widget" native="true">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>341</width>
         <height>80</height>
        </rect>
       </property>
       <widget class="QComboBox" name="year_comboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentText">
         <string>2015</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>2015</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2016</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2017</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2018</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2019</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2020</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2021</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="month_comboBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentText">
         <string>01</string>
        </property>
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="day_comboBox">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="date_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>51</width>
          <height>36</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="year_label">
        <property name="geometry">
         <rect>
          <x>141</x>
          <y>0</y>
          <width>20</width>
          <height>36</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>year</string>
        </property>
       </widget>
       <widget class="QLabel" name="month_label">
        <property name="geometry">
         <rect>
          <x>221</x>
          <y>0</y>
          <width>20</width>
          <height>36</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>month</string>
        </property>
       </widget>
       <widget class="QLabel" name="day_label">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>0</y>
          <width>21</width>
          <height>36</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>741</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Day</string>
        </property>
       </widget>
       <widget class="QPushButton" name="select_date_pushButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>50</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
